### DISCUSSION TITLE
What are some gotchas when using bevy_ecs without bevy_app?

### DISCUSSION THREAD

**Discussion by navneetankur** [bevyengine/bevy#13117] [Q&A]
From a look at documentation I found two.
1. Remember to call Events::update exactly once per update/frame.
2. Remember to call World::clear_trackers once per frame.

Is there anything else? Or are these things listed somewhere already. I can't possibly read and remember the whole bevy_ecs doc.

**Comment by NiklasEi** [comment]
Not specific to usage without `bevy_app`, but one surprise for using `bevy_ecs` outside of Bevy: I wanted to reexport `bevy_ecs` so that users of my library don't have to keep their version in sync. But the derive macros (for `Component`, `Resource` and so on) do not support that.
See bevyengine/bevy#9831
